server/src/main/java/io/druid/server/ClientInfoResource.java

```diff
@@ -83,13 +100,83 @@ public ClientInfoResource(
   @Produces(MediaType.APPLICATION_JSON)
   public Map<String, Object> getDatasource(
```

Comment:
in this case we are giving a 200 OK with empty map. In another case, we wanted to respond with  500 where we're raising exception and jersey will automatically convert it to 500.